app/components/Footer.tsx

```diff
@@ -1,10 +1,165 @@
+import { APP_NAME, Categories, DISCORD_URL, X_URL } from '@/constants';
+import Link from 'next/link';
+
 export const Footer = () => (
-  <div className="flex items-center justify-center space-x-3 px-6 pt-32">
-    <p>
-      built by{' '}
-      <a href="https://swapr.eth.limo" className="text-text-primary-main">
-        Swapr.eth
+  <div className="mt-20 border-t border-surface-surface-2 bg-surface-surface-0 px-6 md:px-10 lg:px-20 xl:px-40">
+    <div className="flex flex-col-reverse items-start justify-between py-10 md:flex-row">
+      <div className="w-full space-y-12 md:w-fit">
+        <div className="hidden space-y-2 md:block">
+          <Logo />
+          <p>Where prediction markets and AI agents meet</p>
+        </div>
+        <PoweredByPartners />
+      </div>
+      <div className="flex space-x-10">
+        <div>
```

Comment:
I don't know about placing the Markets Categories in the footer, it feels out of place.

I like the socials, some other ideas we could place in the footer IMHO are:
* The license this product is under
* a link to our GitHub
* maybe some contact info outside the socials

If we want to focus on the markets, I'd place a floating icon button with a caret pointing up instead of the Markets. This button, on click, would take the user to the page start. As we don't have a very complex sitemap structure I wouldn't place much more links here.